.github/workflows/deploy.yml

```
@@ -152,6 +152,7 @@ jobs:
           GATSBY_FEDS_PRIVACY_ID: ${{ secrets.AIO_FEDS_PRIVACY_ID }}
           GATSBY_SITE_DOMAIN_URL: https://developer-stage.adobe.com
           GATSBY_REDOCLY_KEY: ${{ secrets.REDOCLY_LICENSE_KEY }}
+          GATSBY_API_SPEC_URL: https://api.dev.frame.io/v4/openapi
```

This should probably be staging, instead of dev?
```suggestion
          GATSBY_API_SPEC_URL: https://api.staging.frame.io/v4/openapi
```